data/Ft4us5.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Kirtika",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Newbie in tech| learning web-dev| Exploring Web3",
+    "avatar": "https://github.com/Ft4us5.png",
+    "tags": [
+      "frontend",
+      "open source beginner",
+      "communities"
+      ],
+    "links": [
+      {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/kirtikastwt",
+      "icon": "FaTwitter"
+      },
+      {
+        "name": "Follow me on Github",
+        "url": "https://github.com/Ft4us5",
+        "icon": "FaGithub"
+      },
+      {
+        "name": "Let's connect on LinkeIn",
+        "url": "https://linkedin.com/in/kirtikagoyal",
+        "icon": "FaLinkedin"
+      }
+      ],
+      "milestones":[
+        {
+          "title": "Won at ODCC challenge",
+          "date": "Jan 2023",
+          "color": "purple",
+          "description": "pitched solutions to the problem statements given to me by Bitbns"
+        }
+        ]
+    }
+  
+  
+  
```

Comment:
Please remove these extra lines.
```suggestion
```

Reply:
Hey, David. As mentioned previously, we don't need to point out extra lines fixes. We have a workflow set to fix that after the merge.